build/sync/plan.yml

```diff
@@ -0,0 +1,31 @@
+checks:
+  - type: repo_is_clean
+    params:
+      repo: template
+  - type: repo_is_clean
+    params:
+      repo: plugin
+actions:
+  - paths:
+      - Makefile
+      - .editorconfig
+    actions:
+      - type: overwrite_file
+        params:
+          create: true
+  - paths:
+      - go.mod
```

Comment:
Some of these don't seem like files that we should just overwrite? `go.mod`, `server/configuration.go`, `server/plugin.go`, `server/plugin_test.go`, `webapp/i18n/en.json`, and probably `webapp/package.json` seem likely to contain custom changes over time.

Reply:
@lieut-data Yes I ended up reverting those additions, and manually merged the diffs in the case of `package.json`. The `plan.yml` does not correctly reflect the full migration done here. I'll edit it to be more accurate. This was basically a real-world test run of using the migration tool, and the `plan.yml` file itself is what we should be curating for our different use cases imo.

The `overwrite_file` action is useful, in that I can see the diff in my git diff tool. I don't see us blindly committing those sort of changes without reviewing the diff first. But I agree that some of those files are not the sort of files we want to keep in sync, i.e. `server/configuration.go`.